docs/howto/debugging.md

```diff
@@ -81,6 +82,13 @@ See also [in the "Troubleshooting" section below](#debug-remotely-never-loads).
 [chrome-devtools-device]: https://facebook.github.io/react-native/docs/debugging.html#debugging-on-a-device-with-chrome-developer-tools
 
 
+<div id="react-devtools">
+
+## React Devtools
+
+The standalone version of React DevTools, which is an electron app, can be used to inspect the react-native components of the app. Since the version of react-native we use is `0.59`, the latest version (v4) of react-devtools is not supported, and v3 needs to be installed.  This can be done using the command `npm i -g react-devtools@^3` See [this blog post](https://reactjs.org/blog/2019/08/15/new-react-devtools.html#which-versions-of-react-are-supported) for complete compability information. Refer to [it's npm page](https://www.npmjs.com/package/react-devtools) for instructions on how to use with React Native.
```

Comment:
As discussed in chat, this isn't really about the version of RN we use. Rather, it's that v4 *does not support any released version of React Native*, as of 2020-01.

This also touches on a useful tactic for writing docs: where possible, avoid writing specifics which will change over time, like "we use RN v0.59". When we upgrade, that either becomes one more thing we need to update, or it becomes wrong and potentially misleading. (And we'll have two upgrades *before* the v0.62 upgrade that lets us delete this bit of docs entirely.) Instead, say things that stay true. When that's tricky, you can always add "as of *today's month*" so readers in the future know *when* the statement was true.

Reply:
> As of today's month, we do not support the latest version (v4) of
> react-devtools, and v3 needs to be installed.

Hmm, I guess this point wasn't clear. This sentence is still something that will change over time, and stop being true. What I meant was to write down the actual month that you're writing the text in, like I did in the description above: "as of 2020-01".

But again, this really isn't about what *we* do or don't support. Rather it's that react-devtools v4 *does not support any released version of React Native*, quite beyond our control. So that's the thing we should say.